test/client.test.ts

```diff
@@ -275,11 +275,19 @@ test("fetch: named args", async () => {
     expect(res).toBe("123");
 
     res = await con.fetchOne(`select <str>$a ++ <str>$b`, {
-      a: "123",
       b: "abc",
+      a: "123",
     });
     expect(res).toBe("123abc");
 
+    res = await con.fetchOne(`select <str>$a ++ <str>$b`, {
+      b: "abc",
+      a: "123",
+      c: "def",
+    }).catch((e) => {
```

Comment:
Add a `then` that should raise an exception -- that way we ensure that if we accidentally stop raising an error we detect that.